files/lib/data/conversation/ConversationAction.class.php

```diff
@@ -310,6 +315,8 @@ public function validateMarkAsRead() {
 				$this->parameters['visitTime'] = TIME_NOW;
 			}
 		}
+
+		unset($this->parameters['userID']);
```

Comment:
Side-effect, the `userID` parameter is no longer available in the `finalizeAction`-event.